arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/RaiseAccumulate.kt

```diff
@@ -490,6 +489,15 @@ public open class RaiseAccumulate<Error>(
   public override fun raise(r: Error): Nothing =
     raise.raise(nonEmptyListOf(r))
 
+  @RaiseDSL
+  public inline fun <A, B> Iterable<A>.mapOrAccumulate(
+    transform: RaiseAccumulate<Error>.(A) -> B
+  ): List<B> = raise.mapOrAccumulate(this) { transform(it) }
```

Comment:
microperf: `raise.mapOrAccumulate(this, transform)`

Reply:
```suggestion
  ): List<B> = raise.mapOrAccumulate(this, transform)
```